lib/auth/keystore/keystore_test.go

```diff
@@ -338,3 +342,24 @@ func TestKeyStore(t *testing.T) {
 		})
 	}
 }
+
+func TestLicenseRequirement(t *testing.T) {
+	if os.Getenv("SOFTHSM2_PATH") == "" {
+		t.SkipNow()
```

Comment:
Maybe a message explaining the reason the test is being skipped.

Reply:
good point, added